functions/activate-batch-auto-background.js

```diff
@@ -96,7 +128,7 @@ async function handle(_, db, contract) {
     for(const account of existingAddresses) {
       const count = Math.min(account.balance ?? 0, postRH[account.address] ?? 0)
       holderBalance[account.address] = count
-      for(let i = 0;i < count;i++) {
+      for(let i = 0; i < count + addressBadluckCount[account.address].badLuckCount; i++) {
```

Comment:
We are accounting for uninitialised accounts now but we are not accounting for existing accounts that don't have a bad luck counter